app.rb

```diff
@@ -20,29 +21,29 @@ class Mahah < Sinatra::Base
   end
 
   post '/spaces' do
-    # p params
     Space.create(name: params[:name], description: params[:description], price: params[:price], owner_customer_id: params[:owner_customer_id])
-    
     redirect to "/spaces"
   end
 
   get '/spaces' do
     @spaces = Space.all
-    
     erb :"spaces/index"
-    
   end
 
   get '/bookings/new' do
-
+    @spaces = Space.all
     erb :"bookings/new"
-
   end
 
-  get '/bookings' do
+  # get '/bookings' do
+  #   erb :"bookings/index"
+  # end
+
+  post '/bookings' do
+    @owner = params[:owner]
+    @property = params[:property] 
     erb :"bookings/index"
```

Comment:
This needs a redirect to get '/bookings'
https://learn.co/lessons/sinatra-restful-routes-readme